File: Tests/Tests/Model/ModelObject.m
//
//  ModelObject.m
//  RZImport
//
//  Created by Nick Donaldson on 5/22/14.
//
//

#import "ModelObject.h"

@implementation ModelObject

- (instancetype)initWithID:(NSNumber *)theID
{
    self = [super init];
    if ( self ) {
        _ID = theID;
    }
    return self;
}

@end

File: Tests/Tests/Model/Swift/ModelObject.swift
//
//  ModelObject.swift
//  RZAutoImportTests
//
//  Created by Nick Donaldson on 6/3/14.
//
//

#if os(OSX)
    import Cocoa
#else
    import UIKit
#endif

class ModelObject : NSObject, RZAutoImportable {
    
    let ID: UInt32?
    var lastUpdated: NSDate?
    
    init() {}
    
    init(identifier: UInt32) {
        self.ID = identifier
    }

}
